src/types/plugin-input.ts

```diff
@@ -25,6 +25,7 @@ export const permitGenerationSettingsSchema = T.Object({
   evmNetworkId: T.Number(),
   evmPrivateEncrypted: T.String(),
   permitRequests: T.Array(permitRequestSchema),
+  runId: T.Optional(T.Number()),
```

Comment:
What about this line @whilefoo

Reply:
I'm not sure why this is needed

Reply:
It is used to generate a unique nonce value; please refer to these lines:
https://github.com/hhio618/permit-generation/blob/b147da4e32f90311b4673311024ccc4b244f377b/src/handlers/generate-erc20-permit.ts#L58
https://github.com/hhio618/permit-generation/blob/b147da4e32f90311b4673311024ccc4b244f377b/src/handlers/generate-erc20-permit.ts#L89

Reply:
@whilefoo

Reply:
that makes sense but just using `uuid` to generate a nonce would be sufficient